include/openmc/mesh.h

```diff
@@ -1030,6 +1031,17 @@ class LibMesh : public UnstructuredMesh {
                       //!< elements
   std::vector<int> elem_to_bin_map_; //!< mapping dof indices to bin indices for
                                      //!< active elements
+
+  int extra_element_integer_index_ = -1 ; //!< extra element interger index for element clustering
```

Comment:
why don't we just name this after clustering:
```suggestion
  int cluster_element_integer_index_ = -1 ; //!< extra element integer index for element clustering
```